sparql/allRevisions.q

```diff
@@ -0,0 +1,18 @@
+select *
+where{
+    graph ?record {
+        ?revision rev:describes ex:DocumentId .
+        optional {
+            ?revision rev:isNewRevisionOf ?oldRevision .
+        }
+        ?revision rev:containsRecord ?containedRecord .
+    }
+    filter not exists{
```

Comment:
A revision gets rev:IsNewRevision of all previous revisions. How are this maintained if a document revision replaces another?
To ask in another way; do we need a check to ensure that isNewRevision records doesn't replace each other?